src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -41,23 +73,20 @@ export const WorldsDashboard: React.FC = () => {
               </AdminSidebarSubTitle>
               <AdminSidebarFooter onClickHome={navigateToHome} />
             </AdminSidebar>
-            <AdminShowcase>
+            <AdminShowcase className="WorldsDashboard__worlds">
```

Comment:
`className` added for backwards compatability with pre-existing Admin UI elements upon unification, should probably be removed. 

Ideally the own `.AdminShowcase` class is enough and provides better specificity when used in parent:

```scss
.WorldsDashboard {
  .AdminShowcase {
    // overrides for this specific use of showcase inside worlds dashboard
  }
}
```

Injected class name as in the above example doesn't provide the specificity of `.WorldsDashboard .AdminShowcase` but of a `.WorldsDashboard__worlds`.